CHANGELOG.rst

```diff
@@ -4,6 +4,17 @@ networktocode.nautobot Release Notes
 
 .. contents:: Topics
 
+v2.0.0
```

Comment:
Hit me up and we can document the readme updates.

These are generated using `antsibull-changelog` and I didn't document it.